Brokerages/Alpaca/Markets/Enums/AccountStatus.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Runtime.Serialization;
```

Comment:
I'm assuming these files were sourced from here:
https://github.com/alpacahq/alpaca-trade-api-csharp
I think we should add a `readme.md` file in the `Lean/Brokerages/Alpaca/` folder including the GitHub URL above, just for reference and licensing.

Reply:
We don't mind adding readme.md under Alpaca, but just wanted to make sure we understand the request correctly. I don't see any other broker code under `Brokerages` directory that has readme.md with the original SDK code. Do you still want it for Alpaca specifically, or you are ok with the latest change that just adds the link to the original SDK repository in the source comment?

Reply:
@umitanuki The repo link in the source header comment is totally fine, thanks.

Reply:
👍 Thanks for confirming

Reply:
I think we'll need a statement in each file that was sourced from the `alpaca-trade-api-csharp` repository, preferably referencing the commit hash that they were pulled from, something like (with the correct commit hash):
``` csharp
// Sourced from:  https://github.com/alpacahq/alpaca-trade-api-csharp/tree/45cf16dc1c3ea631ed13f1bd9adee23058ae0bbb
```

I saw another file had the following header:
```
/*
 * The official C# API client for alpaca brokerage
 * https://github.com/alpacahq/alpaca-trade-api-csharp
*/
```
Just want to make sure consumers of this repository are aware of each and every file that is sourced from a separate repository for licensing purposes.

Reply:
Confirmed all files under `./Brokers/Alpaca/Markets` (86 in total) contain the following header:
```
/*
 * The official C# API client for alpaca brokerage
 * Sourced from: https://github.com/alpacahq/alpaca-trade-api-csharp/commit/161b114b4b40d852a14a903bd6e69d26fe637922
*/
```